pkg/config/imports.go

```
@@ -0,0 +1,406 @@
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/charmbracelet/log"
```

golangci-lint (1.62.2)

import "github.com/charmbracelet/log" imported without alias but must be with alias "log" according to config

(importas)